src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -669,8 +679,8 @@ export class ServerView extends DirMixin(PolymerElement) {
       this.monthlyCost = 0;
       this.selectedTab = 'connections';
       this.accessKeySortBy = 'name';
-      /** The direction to sort: 1 == ascending, -1 == descending */
-      this.accessKeySortDirection = 1;
+      /** The direction to sort: 1 == ascending, 0 == no sorting, -1 == descending */
```

Comment:
To avoid peppering magic number constant everywhere, maybe we should have a SortDirection object, like

```
{
  ASCENDING = 1,
  UNSORTED = 0,
  DESCENDING = -1
}
```

Reply:
The initialization is the only place where the value is set with a constant.
And I rely on the assumption this is a number in a bunch of places (adding, comparing, multiplying). I'm not using the value as ids, so I don't think the enum will help much.